layouts/_default/staff-page.html

```diff
@@ -1,3 +1,77 @@
+{{ define "body-class" }}
+  text-[16px] bg-beige font-serif leading-normal text-s-9
+{{ end }}
+
+{{ define "static-head" }}
+  {{ partialCached "header/static-tw.html" . }}
+{{ end }}
+
+{{ define "header" }}
+  <header data-ga-category="header">
+    {{ block "nav" . }}
+      {{ partialCached "tw/nav.html" . }}
+    {{ end }}
+    {{ block "banner" . }}
+      {{ if .Param "banner-active" }}
+        {{ partialCached "tw/banner.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "topper" . }}
+      {{ if .Param "topper-active" }}
+        {{ partialCached "tw/promo-topper.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "promo" . }}
+      {{ partialCached "tw/promo-header.html" . }}
+    {{ end }}
+  </header>
+{{ end }}
+
+{{ define "main" }}
+  <header class="bg-black py-14 px-3 text-white">
+    <div
+      class="mx-auto max-w-[1248px] items-center md:min-h-[110px] gap-y-3 gap-x-5 md:grid md:grid-cols-12"
+    >
+      <h1
+        class="uppercase font-serif text-[45px] leading-none col-span-5 col-start-2 row-span-1 row-start-1 font-black"
+      >
+        {{ .Title }}
+      </h1>
+      <div
+        class="font-raleway col-span-5 col-start-7 row-start-1 text-xl font-medium mt-3 md:mt-0"
```

Comment:
The static-tw.html header doesn't include raleway. If you want to use it, you have to do what about-statecollege.html does:

```
{{ define "static-head" }}
  {{ partial "header/css-tw-rale.html" . }}
  {{ partialCached "header/noscript.html" . }}


  <link
    rel="modulepreload"
    href="{{ partialCached `helper/get-enhancements` . }}"
  />

  {{ partial "header/favicons.html" . }}
  {{ partial "header/feeds.html" . }}
{{ end }}
```

Probably that should be pulled into a partial called header/static-tw-with-raleway.html or something.